src/quafu/qfasm/qfasm_convertor.py

```diff
@@ -26,3 +27,161 @@ def qasm_to_circuit(qasm):
+                                qc.ry(inds[0], paras[0])
+                            elif gatename == "rz":
+                                qc.rz(inds[0], paras[0])
+                            elif gatename == "p":
+                                qc.p(inds[0], paras[0])
+                            elif gatename == "x":
+                                qc.x(inds[0])
+                            elif gatename == "y":
+                                qc.y(inds[0])
+                            elif gatename == "z":
+                                qc.z(inds[0])
+                            elif gatename == "h":
+                                qc.h(inds[0])
+                            elif gatename == "id":
+                                qc.id(inds[0])
+                            elif gatename == "s":
+                                qc.s(inds[0])
+                            elif gatename == "sdg":
+                                qc.sdg(inds[0])
+                            elif gatename == "t":
+                                qc.t(inds[0])
+                            elif gatename == "tdg":
+                                qc.tdg(inds[0])
+                            elif gatename == "sx":
+                                qc.sx(inds[0])
+                            elif gatename == "ccx":
+                                qc.toffoli(inds[0], inds[1], inds[2])
+                            elif gatename == "cswap":
+                                qc.fredkin(inds[0], inds[1], inds[2])
+                            elif gatename == "u1":
+                                qc.rz(inds[0], paras[0])
+                            elif gatename == "u2":
+                                qc.rz(inds[0], paras[1])
+                                qc.ry(inds[0], pi / 2)
+                                qc.rz(inds[0], paras[0])
+                            elif gatename == "u3":
+                                qc.rz(inds[0], paras[2])
+                                qc.ry(inds[0], paras[0])
+                                qc.rz(inds[0], paras[1])
+                            elif gatename == "rxx":
+                                qc.rxx(inds[0], inds[1], paras[0])
+                            elif gatename == "ryy":
+                                qc.ryy(inds[0], inds[1], paras[0])
+                            elif gatename == "rzz":
+                                qc.rzz(inds[0], inds[1], paras[0])
+                            else:
+                                print(
+                                    "Warning: Operations %s may be not supported by QuantumCircuit class currently."
+                                    % gatename
+                                )
+
+    if not qc.measures:
+        qc.measures = dict(zip(range(qc.num), range(qc.num)))
+    if not global_valid:
+        print(
+            "Warning: All operations after measurement will be removed for executing on experiment"
+        )
+
+
+if __name__ == '__main__':
```

Comment:
remember to remove these codes.
if there's any formal test code, add them under tests/ directory ^_^